models/user.js

```diff
@@ -703,16 +704,15 @@ exports.createUserPayment = function(userid, payment, callback = { success: () =
 	let query = `INSERT INTO PaymentMethod (PM_US, PM_Nickname, PM_Name, PM_CardNumber, PM_Expiry, PM_CVC)
 	VALUES (? ,? ,? ,? , ?, ?)`;
 
-	db.query(query,[userid, payment.nickName, payment.name, payment.number, payment.exp, payment.cvc], (err,results) =>
+	db.query(query,[userid, payment.nickName, payment.name, payment.number, dateUtil.convertToMySQLDatetime(payment.exp), payment.cvc], (err,results) =>
```

Comment:
payment exp is not a datetime, even in complex systems, its usually stored as MM/YY